politeiawww/user.go

```diff
@@ -1736,8 +1736,10 @@ func (p *politeiawww) processManageUser(mu *www.ManageUser, adminUser *user.User
 	case www.UserManageReactivate:
 		user.Deactivated = false
 	default:
-		return nil, fmt.Errorf("unsupported user edit action: %v",
-			www.UserManageAction[mu.Action])
+		return nil, www.UserError{
+			ErrorCode:    www.ErrorStatusUnsupportedUserManageAction,
```

Comment:
Changing this to a user error is correct, but it should use the existing `ErrorStatusInvalidUserManageAction` error instead of create a new one.

Also, this `[]string{www.UserManageAction[mu.Action]` is just going to return the string `invalid action`, which is redundant since the error code already tells them its invalid.  I don't think an error context is necessary here because it doesn't provide any additional information to the user.  They only pass up a single action at a time so simply returning the invalid action error code tells them everything they need to know.